index.html

```diff
@@ -35,22 +35,42 @@ <h2>Untrusted Input</h2>
 &lt;/style&gt;</textarea>
     <p><a href="#" id="permalink">Link to this input.</a></p>
 
-    <h2>Live Output</h2>
+    <h2>Sanitizer Configuration</h2>
 
-    <p><strong>Configuration: </strong><code>new Sanitizer(</code><select id="config">
-      <option value="empty">{}</option>
-      <option value="no-attributes">{ "allowAttributes": [] }</option>
-      <option value="no-elements">{ "allowElements": [] }</option>
-      <option value="no-style">{ "dropAttributes": { "style": ["*"] }, "dropElements": [ "style" ] }</option>
-    </select><code>)</code></p>
+    <p>
+      The Sanitizer default configuration is safe and will likely suffice for
+      many applications. Note that all sanitizers guarantee a safe baseline,
+      so even a misconfigured Sanitizer will not allow script execution.
+    </p>
 
-    <p><code>sanitizeToString(<em>input</em>)</code>:</p>
-    <output for="input"></output>
+    <p>
+      <strong>Configuration: </strong>
+      <select id="configselect">
+        <option value="default">default</option>
+        <option value="textonly">text only</option>
+        <option value="style">allow simple styling</option>
+        <option value="misconfigured">misconfigured to allow event handlers</option>
+        <option value="custom">custom (edit below)</option>
+      </select>
+    </p>
+
+    <p id="setconfig">
+      <code>const config = </code>
+      <textarea id="configtext" style="width: 100%;"></textarea>
```

Comment:
Ideally, we'd put the `style` bits into the CSS file.

Reply:
Done.